### DISCUSSION TITLE
As of Friday 19th August 2022, new splines won't load into my react setup

### DISCUSSION THREAD

**Discussion by PeterMain** [splinetool/react-spline#76] [Q&A] [answered]
I'm running NextJS, Recat 16.8.6,  "@splinetool/react-spline": "^2.2.1",  "@splinetool/runtime": "^0.9.64".

https://prod.spline.design/nI3VdvUqSKn2a7VP/scene.splinecode **works**
https://prod.spline.design/mnznnyRsFj5BKEkv/scene.splinecode **doesn't work**

If I update splinetool runtime etc, I need to update react to 17, but then get this web pack error
[message.txt](https://github.com/splinetool/react-spline/files/9394000/message.txt)

**Comment by superguigui** [comment, marked as answer]
Hi @PeterMain 
I was able to load your second link in a new next.js project with those package versions:

![image](https://user-images.githubusercontent.com/400119/185961600-938f9e87-14fa-444e-bf90-102c0e67cf5a.png)
![image](https://user-images.githubusercontent.com/400119/185961833-af22f683-ba81-443d-be06-7cd9cf7ef20b.png)

Do you have custom build tools around next.js ?

The first link is throwing a CORS error for me, I've asked backend about this.

**Comment by PeterMain** [comment]
Thanks, I think the issue was a mixture of old React, old NextJS and some coding ineptitude. Have rebuilt on latest stack and working great.
